### PR TITLE
Timestamp alignment is now part of compaction rules

### PR DESCRIPTION
**Pull request by OfirMos**
Timestamp alignment is now part of compaction rules.

### CODE REVIEW

File: docs/commands/ts.info.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ Each item (per chunk) will contain:
 #### `TS.INFO` Example
 
 ```sql
+
 TS.INFO temperature:2:32
+[b'totalSamples', 3, b'memoryUsage', 4248, b'firstTimestamp', 1, b'lastTimestamp', 5000, b'retentionTime', 0, b'chunkCount', 1, b'chunkSize', 4096, b'chunkType', b'compressed', b'duplicatePolicy', None, b'labels', [], b'sourceKey', None, b'rules', [[b't1_MAX_1000_500', 1000, 500]]]
```
Comment (filipecosta90) [inline docs/commands/ts.info.md:52]: @OfirMos should be pretty-printed if possible

File: src/module.c [hunk 2]
```diff
@@ -105,9 +105,10 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     CompactionRule *rule = series->rules;
     int ruleCount = 0;
     while (rule != NULL) {
-        RedisModule_ReplyWithArray(ctx, 3);
+        RedisModule_ReplyWithArray(ctx, 4);
         RedisModule_ReplyWithString(ctx, rule->destKey);
         RedisModule_ReplyWithLongLong(ctx, rule->bucketDuration);
+        RedisModule_ReplyWithLongLong(ctx, rule->timestampAlignment);
```
Comment (filipecosta90) [inline src/module.c:111]: putting a new info field in the middle might break some clients that parse this based on order...

**Review by filipecosta90 — CHANGES_REQUESTED** [review]
Some small comments. 
I suggest we also explain the on PR and rename it to state that the alignment is now part of the RULE-created time-series. This is behavioral change. Agree @LiorKogan ?

File: docs/commands/ts.info.md (hunk 1 above)
Comment (OfirMos) [reply docs/commands/ts.info.md:52]: @filipecosta90 It was already like this, this PR is just adding it to the ts.info command

**Review by filipecosta90 — APPROVED** [review]
@OfirMos approved, but I believe it's important to emphasize that the fact that timestamp alignment is now part of the rule create keys is a behaviour change from past RTS and should probably be part of RTS v1.8... Agree @LiorKogan @danni-m ?

**Comment by danni-m** [comment]
I agree, this should be considered a new feature

**Comment by OfirMos** [comment]
@filipecosta90 Agree it's going to be released only on 1.8

**Pull request merged**